tests/api/worlds/touch_sensor_force.wbt

```diff
@@ -25,6 +25,10 @@ Robot {
       }
     }
     TouchSensor {
+      lookupTable [
+        -1000 -1000 0
+        1000 1000 0
+      ]
```

Comment:
It would be nice to test also one case where the lookup-table is the default one (not specified) and make sure it returns NULL.

Reply:
Added the test for touch sensor only, let me know if it should be added for the other sensors as well.

Reply:
One is sufficient enough, thank you!